Fw/Types/Serializable.hpp

```diff
@@ -28,8 +28,17 @@ class Serializable {
     using SizeType = FwSizeType;
 
   public:
-    virtual SerializeStatus serialize(SerializeBufferBase& buffer) const = 0;  //!< serialize contents
-    virtual SerializeStatus deserialize(SerializeBufferBase& buffer) = 0;      //!< deserialize to contents
+    virtual SerializeStatus serializeTo(SerializeBufferBase& buffer) const; //!< serialize contents to buffer
+    
+    virtual SerializeStatus deserializeFrom(SerializeBufferBase& buffer); //!< deserialize contents from buffer
```

Comment:
The `serializeTo` and `deserializeFrom` methods should be left pure virtual, no ? 
Are you implementing to call the old names for backwards compatibility?
Looks implementing them is what's causing the infitie recursion  serializeTo() -> serialize() -> serializeTo() ... I don't know of a clean way to do this without past implementations here...